src/server/protocol.ts

```diff
@@ -2588,6 +2590,10 @@ namespace ts.server.protocol {
         insertSpaceBeforeTypeAnnotation?: boolean;
     }
 
+    export interface ServicesSettings {
+        quote: '"' | "'";

```

Comment:
I would say create an enum for this, or even a literal union type that is not the quote. e.g. `"double" | "single"`. VSCode at the end of the day will put this setting in a json file and having ppl wiring `"\""` is not that great.